reference/mongodb/bson/binaryinterface.xml

```diff
@@ -28,6 +28,33 @@
     <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.mongodb-bson-binaryinterface')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])" />
    </classsynopsis>
   </section>
+
+  <section role="changelog">
+   &reftitle.changelog;
+   <para>
+    <informaltable>
+     <tgroup cols="2">
+      <thead>
+       <row>
+        <entry>&Version;</entry>
+        <entry>&Description;</entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry>PECL mongodb 1.15.0</entry>
+        <entry>
+         Return types for methods are declared as tentative on PHP 8.0 and newer,
+         triggering deprecation notices in code that implements this interface
+         without declaring the appropriate return types. The <code>#[ReturnTypeWillChange]</code>
+         attribute can be added to silence the deprecation notice.
```

Comment:
Create a `mongodb.changelog.tentative-return-types` entity in `language-snippets.ent` for the contents of the `<entry>` tag here and reference it to cut down on duplication.

You may also want to link to https://www.php.net/manual/en/language.oop5.inheritance.php#language.oop5.inheritance.internal-classes for more context.

Reply:
Done, thank you!